Extract instruction serialization (#126)
iqm-finland/qiskit-on-iqm: CHANGELOG.rst

Changelog
=========

Version 15.1
============

> * Expose part of instruction serialization as :func:`iqm.qiskit_iqm.iqm_provider.serialize_instructions`.

Reviewer: ```suggestion
* Move a part of circuit serialization into :func:`iqm.qiskit_iqm.iqm_provider._serialize_instructions`.
```
